roles/mariadb/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
-mariadb_keyserver: keyserver.ubuntu.com
+mariadb_keyserver: "hkp://keyserver.ubuntu.com:80"
```

Comment:
Why not `hkps://`?

Reply:
Just copied from Mariadb site but updated to `hkps` and it works 👍